mm/mm_heap/mm_mallinfo.c

```diff
@@ -97,17 +98,29 @@ int mm_mallinfo(FAR struct mm_heap_s *heap, FAR struct mallinfo *info)
 
           if ((node->preceding & MM_ALLOC_BIT) != 0)
             {
+              DEBUGASSERT(node->size >= SIZEOF_MM_ALLOCNODE);
               uordblks += node->size;
             }
           else
             {
+              FAR struct mm_freenode_s *fnode;
+              DEBUGASSERT(node->size >= SIZEOF_MM_FREENODE);
+              fnode = (FAR void *)node;
+              DEBUGASSERT(fnode->blink->flink == fnode);
+              DEBUGASSERT(fnode->blink->size <= fnode->size);
+              DEBUGASSERT(fnode->flink == NULL ||
+                          fnode->flink->blink == fnode);
+              DEBUGASSERT(fnode->flink == NULL ||
+                          fnode->flink->size == 0 ||
+                          fnode->flink->size >= fnode->size);
               ordblks++;
               fordblks += node->size;
               if (node->size > mxordblk)
                 {
                   mxordblk = node->size;
                 }
             }
+          DEBUGASSERT(prev == NULL || prev->size == (node->preceding & ~MM_ALLOC_BIT));
```

Comment:
Coding standard requires that all terminal right braces be followed by a blank line.

Reply:
thank you. fixed.